.travis.yml

```diff
@@ -12,4 +12,5 @@ script:
     - go test -v ./...
     - chmod u+x ./official-git/run-tests.sh
     - GIT_SKIP_TESTS='t0008.30[5-9] t0008.310 t0008.321 t0008.323 t0008.37[0-9] t0008.38[0-7] t0008.39[1-2]' ./official-git/run-tests.sh t0000-basic.sh t0004-unwritable.sh t0007-git-var.sh t0008-ignores.sh t0010-racy-git.sh t0062-revision-walking.sh t0070-fundamental.sh t0081-line-buffer.sh t1009-read-tree-new-index.sh t1304-default-acl.sh t2100-update-cache-badpath.sh t4113-apply-ending.sh t4123-apply-shrink.sh t5705-clone-2gb.sh t7062-wtstatus-ignorecase.sh t7511-status-index.sh
+    - ./dgit clone https://github.com/driusan/dgit.git /tmp/dgit.$$
```

Comment:
I don't understand how this works without having done a go build first,  but it seems to actually be running in the travis logs, so I'll take it